### DISCUSSION TITLE
关于web-worker demo的问题

### DISCUSSION THREAD

**Discussion by WhatIsSpark** [Tencent/libpag#1905] [Q&A] [answered]
使用web-worker https://github.com/libpag/pag-web/blob/main/pages/web-worker.html
是否有一种方式可以像 set-composition demo一样替换pag上个动画,
https://github.com/libpag/pag-web/blob/main/pages/set-composition.html

**Comment by zhalice2011** [comment, marked as answer]
你好, 当前的 web-worker  版本没有支持 set-composition 方法

方便提供使用场景demo吗, 我们可以考虑后续支持
